formserver/src/main/java/com/diyasylum/formserver/serializers/EncryptedMessageSerializer.java

```diff
@@ -0,0 +1,54 @@
+package com.diyasylum.encryption.serializers;
+
+import com.diyasylum.encryption.models.EncryptedMessage;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.TreeNode;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.node.TextNode;
+import java.io.IOException;
+import org.apache.commons.codec.binary.Base64;
+import org.springframework.boot.jackson.JsonComponent;
+
+@JsonComponent
+public class EncryptedMessageSerializer {
+
+  private static Base64 base64 = new Base64();
```

Comment:
Dont think you need this. The methods you are calling are static

Reply:
Originally I couldn't get it to compile with the static methods but now it's working just fine.